custom_components/daikinone/daikinone.py

```diff
@@ -289,7 +289,7 @@ def __map_equipment(self, payload: DaikinDeviceDataResponse) -> dict[str, Daikin
         if payload.data["ctAHUnitType"] < 255:
             model = payload.data["ctAHModelNoCharacter1_15"].strip()
             serial = payload.data["ctAHSerialNoCharacter1_15"].strip()
-            eid = f"{model}-{serial}"
```

Comment:
@wirecuttr if you can revert the changes here, would be great to merge, thanks!

Reply:
I'll do my best :). I'm pretty new to al of this. I merged two branches on my end, not realizing that it impacts the pull request!